2021/Dundee/OMERO.figure-workshop/index.html

```diff
@@ -0,0 +1,252 @@
+                -webkit-hyphens: manual;
+                -moz-hyphens: manual;
+                hyphens: manual;
+            }
+            .reveal h1 {
+                font-size: 2.5em;
+            }
+            .ome-theme body {
+                background-image: none;
+            }
+        </style>
+
+    </head>
+
+    <body>
+
+        <div class="reveal">
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section data-state="ome-theme">
+                    <h1>Research Data Management</h1>
+                    <h2>OMERO.figure workshop</h2>
+                    <img style="width:150px;" src="images/ome-icon-opacity.png"/>
+                    <p>2021 OME Community Meeting</p>
+                    <p>
+                        9th June 2021 | Will Moore
+                    </p>
+                </section>
+                
+                <section data-state="ome-theme">
+                    <h2>Outline</h2>
+                    <ul>
+                        <li>Intro: OME & OMERO</li>
+                        <li>Workshop overview</li>
+                        <li>Workshop...</li>
+                        <li>Breakout Rooms
+                            <ul>
+                                <li>OMERO.figure UI</li>
+                                <li>Scripting (Python & JavaScript)</li>
+                            </ul>
+                        </li>
+
+                    </ul>
+                </section>
+
+                
+                <section data-state="ome-theme" data-background="none">
+                    <h2>OME: Interoperability</h2>
+
+                    <img src="images/ome-informatics.png">
+                </section>
+
+
+                <section data-background-size="971px" data-background="images/omero-platform-bg-0.png">
+                    <h2>OMERO is a server with clients</h2>
+                    <div style="width: 600px; height: 600px"></div>
+                </section>
+
+                <section data-background="images/webclient-plate2.png">
+                </section>
+
+                <section data-background-size="971px" data-background="images/omero-platform-bg-4.png">
+                    <h2>OMERO API</h2>
+
+                    <div style="height: 350px">
+                    Python | Java | Matlab | R | JavaScript
+                    </div>
+                </section>
+
+                <section data-state="ome-theme" style="width:120%; left:-60%">
+                    <h2><a target="_blank" href="https://idr.openmicroscopy.org/">idr.openmicroscopy.org</a></h2>
+                    <img src="images/idr.png" width="1900px"/>
+                </section>
+
+                <section data-state="ome-theme" style="color: white" data-background="images/idr0079-pLLP_in_whole_embryo.png">
+                    <h2 style="color: white">Today's workshop:</h2>
```

Comment:
This page almost makes me think to generate a reveal.js for each study ;)

cc: @sbesson @francesw